core/shared/src/test/scala/fs2/StreamSpec.scala

```diff
@@ -1614,6 +1615,7 @@ class StreamSpec extends Fs2Spec {
             case Some((hd, tl)) => Pull.eval(IO.never)
           }
           .stream
+          .interruptScope
```

Comment:
Note b/c pulls no longer introduce scopes, I had to add this manually here. I think this is the right tradeoff though.

Reply:
agree